### ISSUE TITLE
Changelog for pg8000

### ISSUE THREAD

**Issue by rrueth** [dependencies-io/support#60]
```
pg8000
This dependency is located in requirements.txt and was updated from "1.11.0" to "1.12.3".

We didn't find any content for 1.12.0, 1.12.1, 1.12.2, or 1.12.3. Feel free to open an issue at https://github.com/dependencies-io/support to suggest any improvements.
```

The changelog is in the README.adoc in the `Release Notes` section: https://github.com/mfenniak/pg8000/blob/master/README.adoc

**Comment by davegaeddert** [comment]
May have to look into this further, but we use pandoc to convert everything to markdown and I don't see support for AsciiDoc (at least not out of the box?).

**Comment by rrueth** [comment]
Yeah, it looks like Pandoc doesn't read AsciiDoc, but does write AsciiDoc.  I found this if it's helpful:

https://gist.github.com/cheungnj/38becf045654119f96c87db829f1be8e

**Comment by davegaeddert** [comment]
Yeah that's about where I ended up too. I'll see if we can consider adding asciidoctor into the toolchain, though I have to admit this is the first time I've seen AsciiDoc come up.
